app/scripts/lib/setupSentry.js

```diff
@@ -238,7 +238,7 @@ function getSentryEnvironment() {
 
 function getSentryTarget() {
   if (
-    !getManifestFlags().sentry?.forceEnable ||
+    getManifestFlags().sentry?.forceEnable === false ||
```

Comment:
Alternatively, we could do something like:

```
if (process.env.IN_TEST && (!SENTRY_DSN_DEV || !getManifestFlags().sentry?.forceEnable))
```

This would accomplish the same goal (I assume we only wanted this fake DSN during e2e tests), and would avoid treating `undefined` differently from `false`. Having that be a meaningful distinction seems confusing.

Reply:
Yes, that should do the trick, because if you're `IN_TEST`, `forceEnable` will not be `undefined`